### ISSUE TITLE
shitcoins.club ATMs

### ISSUE THREAD

**Issue by matkoniecz** [osmlab/name-suggestion-index#9542]
Yes, it is not an elaborate joke. 

See https://shitcoins.club/pl/mapa-bitomatow

(though there is about 90 of them - is it enough to qualify for NSI?)

**Comment by Cj-Malone** [comment]
We have entries with less than that, so yeah, if you can get a wikidata entry we can add it.

**Comment by Cj-Malone** [comment]
Wikidata isn't actually required to be in here, however it is required for the entry to be displayed in iD. I don't know if StreetComplete should have the same requirement.

**Comment by mnalis** [comment]
Yes, I've seen few of those in Croatia too. Those should always have [currency:XBT=yes](https://wiki.openstreetmap.org/wiki/Key:currency:XBT) in NSI, right? Wikidata is always good (at least to specify website, but logos etc. can be added too)

**Comment by matkoniecz** [comment]
> I don't know if StreetComplete should have the same requirement.

right now it has no requirement as this data is not used right now (spotted that this minor brand is missing while looking at data)
